WalletWasabi.Packager/Program.cs

```diff
@@ -187,8 +153,9 @@ private static void ReduceOnions()
 				{
 					var verString = userAgent.Substring(userAgent.IndexOf("Satoshi:") + 8, 4);
 					var ver = new Version(verString);
+					bool condition = currentOnions is null ? true : currentOnions.Contains(node.Name);
```

Comment:
bool addToResult = currentOnions is null ? true : currentOnions.Contains(node.Name);